tests/testthat/test-convert_es.R

```diff
@@ -0,0 +1,275 @@
+#' Loads in a .RData file that has pre-made variables (from the vignette) and tests them
+#' with manually created data.frames.
+#' 
+#' TODO: Finetune the tolerance values
+
+context("convert_es - testing vignette examples")
+
+library(psychmeta)
+
+###### Make sure this is correct!
+load("~/GitHub/psychmeta/tests/testthat/RData_testfiles/test-convert_es-actual.RData")
```

Comment:
See here for how to load data in tests
http://r-pkgs.had.co.nz/data.html#other-data
https://stackoverflow.com/questions/24823353/testthat-in-r-sourcing-in-tested-files
https://stackoverflow.com/questions/24823353/testthat-in-r-sourcing-in-tested-files